src/curated-corpus/components/ScheduleSummaryConnector/ScheduleSummaryConnector.tsx

```diff
@@ -112,19 +108,23 @@ export const ScheduleSummaryConnector: React.FC<
 
   // Finally, let's render everything onto the screen.
   return (
-    <div className={classes.root}>
-      <h4>Scheduled for {displayDate}</h4>
+    <>
       {!data && <HandleApiResponse loading={loading} error={error} />}
-      {data && data.getScheduledCorpusItems[0]?.items.length < 1 && (
-        <Typography className={classes.heading} variant="h4">
-          No results
-        </Typography>
-      )}
+      {data &&
+        (data.getScheduledCorpusItems.length < 1 ||
+          data.getScheduledCorpusItems[0]?.items.length < 1) && (
+          <Typography className={classes.heading} variant="h4">
+            No stories have been scheduled for this date yet.
+          </Typography>
+        )}
       {data && data.getScheduledCorpusItems[0]?.items.length > 0 && (
         <>
           <Typography className={classes.heading} variant="h4">
-            ({data.getScheduledCorpusItems[0]?.syndicatedCount}/
-            {data.getScheduledCorpusItems[0]?.totalCount} syndicated)
+            {data.getScheduledCorpusItems[0]?.totalCount}{' '}
+            {data.getScheduledCorpusItems[0]?.totalCount === 1
+              ? 'story'
+              : 'stories'}
+            , {data.getScheduledCorpusItems[0]?.syndicatedCount} syndicated
```

Comment:
Should we pull out these conditions into separate variables and put them above the `render()`? I think it would help a bit with readability

Reply:
@Herraj, it would help with readability in the component, but the vars would have to go into `onCompleted` as they'd be undefined without data being resolved first, and then TypeScript complains that they need a value anyway before that if they go into the component, so +4 lines of code to improve readability here. I was thinking about a follow-up PR but decided to leave as is for now.